packages/ui/src/preloader/Preloader.tsx

```diff
@@ -0,0 +1,13 @@
+import { Skeleton as MuiSkeleton, SkeletonProps } from '@material-ui/lab';
+import styled from 'styled-components';
+
+const StyledSkeleton = styled(MuiSkeleton)`
+  &.MuiSkeleton-text {
+    transform: scale(1, 1);
+    border-radius: 3px;
+  }
+`;
+
+export function Skeleton({ ...props }: SkeletonProps): JSX.Element {
+  return <StyledSkeleton {...props} />;
+}
```

Comment:
Thanks @timurhamitov. Yes I got it and overrided with theme ui. Yes there is issue with `scaling` props here on skeleton component, which is 60% scale from outer component. If Skeleton set 30 px height, Skeleton will scale up to 18px. I want to avoid misunderstanding measurement here
<img width="934" alt="image" src="https://github.com/superdispatch/web-ui/assets/118124821/f5679711-0e5a-4943-a6cc-d0b8644fe11b">